apollo/__main__.py

```diff
@@ -96,6 +106,9 @@ def add_template_args(my_parser, default_template):
     add_cassandra_args(source2bags_parser)
     add_df_args(source2bags_parser)
     add_repartitioner_arg(source2bags_parser)
+    source2bags_parser.add_argument(
```

Comment:
Looks like we should move it to sourced-ml and re-use here

Reply:
on it, will add to https://github.com/src-d/ml/pull/285